web-console/knikubevirt/migrate-vm/migrate-vm.md

```diff
@@ -4,13 +4,15 @@ Clicking the action button next to any Virtual Machine in the List will present
 
 ![Selecting a VM](img/1-0-list.png)
 
-A modal will be presented where the user can choose which Node the VM should be migrated to.
+When `migrate` is selected the user is presented with a dialog the requires them to confirm the migration. 
```

Comment:
small nit on wording ... i think "dialog the requires" should be "dialog that requires"